src/main/scala/viper/gobra/translator/encodings/channels/ChannelEncoding.scala

```diff
@@ -68,7 +67,7 @@ class ChannelEncoding extends LeafTypeEncoding {
           )
 
           // exhale [c].RecvGivenPerm()()
-          recvGivenPermInst = getChannelInvariantAccess(channel, recvGivenPerm, Vector())(exp.info)
+          recvGivenPermInst = getChannelInvariantAccess(channel, recvGivenPerm, Vector.empty, Vector.empty)(exp.info)
```

Comment:
using `Vector.empty` instead of `Vector()` sounds like a good idea! I wonder, does it actually save an allocation, or is there any specific optimisation for calling the constructor with an empty list?

Reply:
I've actually no idea. My motivation was to slightly help readability